notebooks/geocat/00-Overview.ipynb

```diff
@@ -16,7 +23,7 @@
     }
    },
    "source": [
-    "# What is GeoCAT?\n",
+    "## What is GeoCAT?\n",
```

Comment:
You should be able to keep one top level header per file and not mess up any of the toctrees

Reply:
Yes, and that is the "# GeoCAT Overview" header in this NB. Thanks!